zorg/buildbot/builders/ClangBuilder.py

```diff
@@ -154,7 +154,7 @@ def getClangCMakeBuildFactory(
             testsuite_flags=None,
             submitURL=None,
             testerName=None,
-
+            testWithLNT=True,
```

Comment:
`test` could mean `ninja check` or `llvm-test-suite`, perhaps change this to `testsuite_use_lnt` to match the other parameter? Then it's more clear what it changes.